### PR TITLE
Revert changes from PR3602 to remove usePreviewSdk

### PR DESCRIPTION
**Pull request by dmonroym**
We're moving the "Use previews of .NET Core SDK" option to the new Preview Tab

This is dependent on [this](https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/VS/pullrequest/173450) PR which moves it.

Related to internal bug [818730](https://dev.azure.com/devdiv/DevDiv/_workitems/edit/818730)

### CODE REVIEW

File: src/Microsoft.VisualStudio.Editors/PublicAPI.Shipped.txt [hunk 1]
```diff
@@ -261,14 +261,11 @@ Microsoft.VisualStudio.Editors.MyApplication.MyApplicationProperties.SingleInsta
 Microsoft.VisualStudio.Editors.MyApplication.MyApplicationProperties.SplashScreen() -> String
 Microsoft.VisualStudio.Editors.MyApplication.MyApplicationProperties.SplashScreen(value As String) -> Void
 Microsoft.VisualStudio.Editors.OptionPages.GeneralOptions
-Microsoft.VisualStudio.Editors.OptionPages.GeneralOptions.CanChangeUsePreviewSdk() -> Boolean
```
Comment (jjmew) [inline src/Microsoft.VisualStudio.Editors/PublicAPI.Shipped.txt]: I am not familiar with how this file gets used but this looks like we removed a public API. Is that a public API as in externals (to Microsoft) can access it? David - Do you know?

**Comment by jjmew** [comment]
@jmarolf is added to the review. #Closed

File: src/Microsoft.VisualStudio.Editors/PublicAPI.Shipped.txt (hunk 1 above)
Comment (jjmew) [reply src/Microsoft.VisualStudio.Editors/PublicAPI.Shipped.txt]: Maybe Jonathon know @ 

---
In reply to: [272756491](https://github.com/dotnet/project-system/pull/4696#discussion_r272756491) [](ancestors = 272756491)

Comment (Pilchie) [reply src/Microsoft.VisualStudio.Editors/PublicAPI.Shipped.txt]: I wouldn't worry about it - this was new in 15.9 anyway.

**Pull request merged**